akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -125,7 +159,11 @@ class CircuitBreaker(
    */
   @inline
   private[this] def swapState(oldState: State, newState: State): Boolean =
-    Unsafe.instance.compareAndSwapObject(this, AbstractCircuitBreaker.stateOffset, oldState, newState)
+    Unsafe.instance.compareAndSwapObject(
+      this,
+      AbstractCircuitBreaker.stateOffset,
+      oldState,
+      newState)
```

Comment:
these re-formatting make it difficult to review, and I can't say that this reads better